kdb/kdb.go

```diff
@@ -66,6 +66,7 @@ func (e *kdbC) Close() error {
 }
 
 // Get retrieves parentKey and all Keys beneath it.
+// Returns true if Keys have been loaded or updated.
```

Comment:
How is it possible to know the difference between doing nothing (return code 0) and error (return code -1)?

Reply:
The second return argument `error` is nil if `return code == 0` and `!= nil` if  `-1`

Reply:
Please also document second argument.